.circleci/config.yml

```diff
@@ -36,7 +36,8 @@ workflows:
   version: 2
   ci:
     jobs:
-      - build
+      - build:
+          context: fx-libraries
```

Comment:
every job will need the context, otherwise it can't clone I think

Reply:
It seems the r/o ssh key circleci uses to clone the repo does not exist in the github repo settings. I'm looking into it.